### PR TITLE
README reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by c72578** [comment]
@comradekingu Please consider, that README is a plain text file, no Markdown.

File: README [hunk 1]
```diff
@@ -23,8 +26,8 @@ class reference.
 Easily-installable prebuilt binaries for Windows and macOS are available from
 https://poedit.net/download.
 
-Official binaries for Linux are available as a Snap at https://snapcraft.io/poedit
-Most Linux distributions also include native Poedit packages.
+Official binaries for Linux are available as a Snap package at https://snapcraft.io/poedit
+Most Linux|GNU distributions also include native Poedit packages.
```
Comment (vslavik) [inline README:30]: The actual spelling of that is "GNU/Linux", not this "Linux|GNU" nonsense, and I am not going to engage in this kind of political speech here.

Comment (comradekingu) [reply README:30]: That was what I was trying to avoid. All in good fun. Do you want me to remove it?

File: README [hunk 2]
```diff
@@ -33,10 +36,10 @@ Most Linux distributions also include native Poedit packages.
 Requirements:
 
   * Boost
-  * Unicode build of wxWidgets library, version >= 3.0.3 (see http://www.wxwidgets.org)
+  * Unicode build of the wxWidgets library. (see https://www.wxwidgets.org), version >= 3.0.3
```
Comment (vslavik) [inline README:39]: That added "." doesn't belong there.

**Comment by vslavik** [comment]
In addition to this being a disorganized mess of multiple commits (please see https://chris.beams.io/posts/git-commit/ — one commit per one thing, clearly explained), it is unclear what this PR's goal even *is* (see https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests). As far as I could discern, it improves some trivia, adds some FSF agenda and does some whitespace changes that don't seem to have a purpose; it doesn't seem to have "reworked" the README file. 

In short, I don't quite understand why this should be applied.

**Comment by comradekingu** [comment]
OK, I am not trying to force your hand.
Have a continued nice day.

**Comment by vslavik** [comment]
> OK, I am not trying to force your hand.

It's not about that. It's about providing at least basic *rationale* for a change, and following basic hygiene. You can't reasonably expect any project to merge in any random PR with no description or explanation just because it was submitted.
